docs/modules/ROOT/pages/index.adoc

```diff
@@ -425,24 +434,13 @@ The following example uses Mutiny reactive types.
 If you are not familiar with Mutiny, check xref:mutiny-primer.adoc[Mutiny - an intuitive reactive programming library].
 ====
 
-Add the following dependency to your `pom.xml`:
-
-[source,xml]
-----
-<dependencies>
-    <dependency>
-        <groupId>io.quarkus</groupId>
-        <artifactId>quarkus-resteasy-mutiny</artifactId>
-    </dependency>
-</dependencies>
-----
-
```

Comment:
This is not needed anymore. RESTEasy Reactive comes with Mutiny support by default.